integr8ly/integreatly-operator
# MGDAPI-2890 - Upgrade keycloak-operator to v17.0.0

# Issue link
[MGDAPI-2890](https://issues.redhat.com/browse/MGDAPI-2890)

# What
- Added manifests from keycloak-operator v17.0.0
- Removed hard-coded replacement of JDK images

# Verification steps
- Verify upgrading RHOAM using this branch installs keycloak-operator v17.0.0
  - quay.io/acatterm/managed-api-service-index:1.19.0 (built from master)
  - quay.io/acatterm/managed-api-service-index:1.25.0 (built from this PR) 
- Check keycloak images are replaced with latest version - see these images from the catalog - [keycloak](https://catalog.redhat.com/software/containers/rh-sso-7/sso75-openshift-rhel8/611418806e1e42ca4d6decf1?push_date=1646841023000&architecture=amd64&tag=all), [init-container](https://catalog.redhat.com/software/containers/rh-sso-7/sso7-rhel8-init-container/6160610d5ac6c12504902068?tag=all)
